### ISSUE TITLE
[BUG]The Logs page in the UI is linked to two js files that do not exist

### ISSUE THREAD

**Issue by weizhuangzhi** [bunkerity/bunkerweb#614]
**Description**
```
<script type="module" src="./js/utils/fr.js"></script>
......
<script type="module" src="./js/datepicker/datepicker.js"></script>

```
Chrome：404 Not Found

Not sure if these two files are missing or useless?

**Comment by TheophileDiot** [comment]
Hi, thank you for this issue, it will be fixed in the next release.

**Comment by fl0ppy-d1sk** [comment]
Hello @weizhuangzhi,

It should be fixed in the last [1.5.2 version](https://github.com/bunkerity/bunkerweb/releases/tag/v1.5.2). Feel free to open a new issue if needed.
